packages/loader/container-loader/src/protocol.ts

```diff
@@ -75,11 +75,13 @@ export class ProtocolHandler extends ProtocolOpHandler implements IProtocolHandl
 		message: ISequencedDocumentMessage,
 		local: boolean,
 	): IProcessMessageResult {
-		const client: ILocalSequencedClient | undefined = this.quorum.getMember(message.clientId);
-
 		// Check and report if we're getting messages from a clientId that we previously
 		// flagged as shouldHaveLeft, or from a client that's not in the quorum but should be
 		if (message.clientId != null) {
+			const client: ILocalSequencedClient | undefined = this.quorum.getMember(
```

Comment:
Good stuff - side note, seems like the errors below should be asserts.

Reply:
possibly. not trying to do more than just prepare for the incoming protocol changes.